djangocms_snippet/cms_config.py

```diff
@@ -0,0 +1,17 @@
+from cms.app_base import CMSAppConfig, CMSAppExtension
+
+from djangocms_versioning.datastructures import VersionableItem, default_copy
+
+from .models import Snippet
+
+
+class SnippetCMSAppConfig(CMSAppConfig):
+    djangocms_versioning_enabled = True
```

Comment:
This needs to be optional similar to Alias: https://github.com/divio/djangocms-alias/blob/master/djangocms_alias/cms_config.py#L26